src/js/iconify.js

```diff
@@ -56,6 +66,13 @@ function findElementToSyndicate (element) {
 		element.tagName.toUpperCase() !== 'FORM'
 		&& element.tagName.toUpperCase() !== 'BUTTON';
 
+
+	// data-syndication-content-id should be used on the element we want to syndicate
+	// This means there is no need to search for the element
+	if (element.hasAttribute('data-syndication-content-id') && SWITCH_SELECTOR && elementIsNotFormOrButton) {
```

Comment:
**Praise:** Love the use of a dedicated data attribute to use as a query selector.